deck-service/src/main/java/magicthebuilder/deckservice/dto/MultipleCardDto.java

```diff
@@ -0,0 +1,20 @@
+package magicthebuilder.deckservice.dto;
+
+import com.sun.istack.NotNull;
+import lombok.*;
+
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class MultipleCardDto {
+    @NotNull
+    private String cardId;
+
+    @NotNull
+    private int amount;
```

Comment:
Just curious. int cannot be ever null. Is this doing something (e.g. Is it required by some library)?

Reply:
Fixed in ead6168